src/patternfly/components/Sequence/sequence.scss

```diff
@@ -0,0 +1,309 @@
+
+  // Content
+  --pf-c-sequence__content--MinHeight: var(--pf-global--spacer--xl);
+  --pf-c-sequence__content--MaxWidth: #{pf-size-prem(140px)};
+
+  // Content Title Color
+  --pf-c-sequence__content-title--Color: var(--pf-global--Color--100);
+  --pf-c-sequence__content-title--hover--Color: var(--pf-global--primary-color--100);
+  --pf-c-sequence__content-title--active--Color: var(--pf-global--primary-color--100);
+  --pf-c-sequence__content-title--focus--Color: var(--pf-global--primary-color--100);
+  --pf-c-sequence__content-title--m-current--Color: var(--pf-global--primary-color--100);
+
+  // Content Title Font
+  --pf-c-sequence__content-title--FontSize: var(--pf-global--FontSize--md);
+  --pf-c-sequence__content-title--FontWeight: var(--pf-global--FontWeight--reguar);
+  --pf-c-sequence__content-title--active--FontWeight: var(--pf-global--FontWeight--semi-bold);
+  --pf-c-sequence__content-title--focus--FontWeight: var(--pf-global--FontWeight--semi-bold);
+  --pf-c-sequence__content-title--m-current--FontWeight: var(--pf-global--FontWeight--semi-bold);
+
+
+  // Content Status Color
+  --pf-c-sequence__content-status--Color: var(--pf-global--Color--200);
+
+  // Content Status Font
+  --pf-c-sequence__content-status--FontSize: var(--pf-global--FontSize--xs);
+  --pf-c-sequence__content-status--FontWeight: var(--pf-global--FontWeight--regular);
+
+  display: flex;
+  justify-content: center;
+  height: var(--pf-c-sequence--Height);
+  overflow-y: hidden;
+
+  // remove default border styling
+  button {
+    background-color: transparent;
+    border-top: 0;
+    border-bottom: 0;
+
+    &:focus {
+      outline: 0;
+    }
+  }
+
+  // remove scrollbar works on Chrome and Safari
+  ::-webkit-scrollbar {
+    display: none;
+    width: 0;
+  }
+}
+
+.pf-c-sequence__control {
+  padding-top: var(--pf-c-sequence__control--PaddingTop);
+  padding-bottom: var(--pf-c-sequence__control--PaddingBottom);
+
+  &::before {
+    content: "";
+    border-top: var(--pf-c-sequence__control--BorderTopWidth) solid var(--pf-c-sequence__control--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--BorderBottomWidth) solid var(--pf-c-sequence__control--BorderBottomColor);
+  }
+
+  &.pf-m-hover,
+  &:hover {
+    border-top: var(--pf-c-sequence__control--hover--BorderTopWidth) solid var(--pf-c-sequence__control--hover--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--hover--BorderBottomWidth) solid var(--pf-c-sequence__control--hover--BorderBottomColor);
+  }
+
+  &.pf-m-focus,
+  &:focus {
+    border-top: var(--pf-c-sequence__control--focus--BorderTopWidth) solid var(--pf-c-sequence__control--focus--BorderTopColor);
+    border-bottom: var(--pf-c-sequence__control--focus--BorderBottomWidth) solid var(--pf-c-sequence__control--focus--BorderBottomColor);
+  }
+
+  @media screen and (min-width: $pf-global--breakpoint--xl) {
+    display: none;
+  }
+}
+
+.pf-c-sequence__control-m-start {
```

Comment:
this should be `.pf-m-start` (and `.pf-m-end` below), and nested inside of `.pf-c-sequence__control`